stdlib/public/Concurrency/Task.swift

```diff
@@ -591,7 +591,7 @@ public func withUnsafeCurrentTask<T>(body: (UnsafeCurrentTask?) throws -> T) ret
   // FIXME: This retain seems pretty wrong, however if we don't we WILL crash
   //        with "destroying a task that never completed" in the task's destroy.
   //        How do we solve this properly?
-  _swiftRetain(_task)
+  Builtin.retain(_task)
```

Comment:
Oh i see, thank you.

Reply:
The remaining need for these is just that `Builtin.currentAsyncTask()` is returning an unretained pointer.  It probably simply shouldn't, but I wanted to leave that for a different PR.

Reply:
Yeah that probably makes sense 👍